content/en/continuous_integration/setup_pipelines/customspans.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Custom spans
+kind: documentation
+further_reading:
+    - link: "/continuous_integration/setup_pipelines/customspans/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+Custom spans provide a way to trace individual commands in your CI pipelines, allowing you to measure the time your command takes without taking into account any setup or teardown actions that the job might have (for example, downloading Docker images or waiting for an available node in a Kubernetes-based infrastructure). These spans appear as part of the pipeline's trace:
+
+{{< img src="ci/ci-custom-spans.png" alt="Details for a single pipeline with custom spans" style="width:100%;">}}
+
+## Compatibility
+
+Custom spans work with the following CI providers:
+
+- Jenkins with Datadog plugin >= v3.2.0
+- CircleCI
+
+## Installing the Datadog CI CLI
+
+Install the [`datadog-ci`][1] CLI globally using `npm`:
+
+{{< code-block lang="bash" >}}
+npm install -g @datadog/datadog-ci
+{{< /code-block >}}
+
+## Tracing a command
+
+To trace a command, run the following:
+
+{{< code-block lang="bash" >}}
+datadog-ci trace --name <name> <command>
+{{< /code-block >}}
+
+Specify a valid [Datadog API key][2] in the `DATADOG_API_KEY` environment variable. For example:
+
+{{< site-region region="us" >}}
+{{< code-block lang="bash" >}}
+DATADOG_API_KEY=<api_key> datadog-ci trace \
+  --service "Greet" \
+  echo "Hello World"
+{{< /code-block >}}
+{{< /site-region >}}
+{{< site-region region="eu" >}}
+{{< code-block lang="bash" >}}
+DATADOG_API_KEY=<api_key> DATADOG_SITE=datadoghq.eu datadog-ci trace \
+  --service "Greet" \
+  echo "Hello World"
+{{< /code-block >}}
+{{< /site-region >}}
+
+## Configuration settings
+
+This is the full list of options available when using the `datadog-ci trace` command:
+
+`--name`
+: Display name of the custom span.<br/>
+**Default**: same value as `<command>`<br/>
+**Example**: `Wait for DB to be reachable`
```

Comment:
do we want to show this between `"` ? 
Plus, is this the kind of value we want to encourage the user to use? This goes to operation name, doesn't it? These values are usually more like `grpc.client` or `junit.test`. Not that I don't think it wouldn't fit, just pointing it out.

Reply:
We are not using `"` in the junit page. I can add them in both places if you want.

This is not the operations name (which will be `<ciprovider>.custom`). This maps to the resource name.